custom/covid/management/commands/update_cases.py

```diff
@@ -28,6 +28,7 @@ def add_arguments(self, parser):
         parser.add_argument('domain')
         parser.add_argument('case_type')
         parser.add_argument('--username', type=str, default=None)
+        parser.add_argument('--active-location', type=str, default=None)
```

Comment:
It would be great if this were only added to the new command (since it's not used in the old ones). It can probably also be shortened to `location` since there is not reason it has to be "active." It can be any location in the domain. Alternatively you could add location filters to the old commands, but the current situation of accepting and ignoring the argument is a bit confusing.